packages/server/src/index.js

```diff
@@ -10,23 +10,15 @@ import morgan from 'morgan'
 import { createServer } from '@promster/server'
 import { createMiddleware, signalIsUp } from '@promster/express'
 
-import errorHandler from './helpers/error-handler'
 import routes from './routes'
-import db from './models'
+import errorHandler from './helpers/error-handler'
+import checkConnection from './database/check-connection'
 
+// Load env variables and check DB connection
 dotenv.config()
+checkConnection().then(console.log)
```

Comment:
Wondering, would it be better to only start the express app if we were able to connect to the db?